index.html

```diff
@@ -4,27 +4,26 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <link rel="stylesheet" href="styles.css"/>
-  <link rel="stylesheet" href="queries.css"/>
+  <link rel="stylesheet" href="./styles.css"/>
   <title>Portfolio</title>
 </head>
 <body>
   <main class="container">
   
     <header class="header">
-      <nav class="desktop-navbar">
-        <div class="desktop-nav">
-          <a href="#" target="_parent" class="logo">Tanveer Ahmad</a>          
+      <nav class="desktop-navbar d-flex">
+        <div class="desktop-nav d-flex">
+          <a href="#" target="_parent" class="logo f-size">Tanveer Ahmad</a>          
```

Comment:
- [ ] Website is still not responsive below 768px width remaining the same which is not good for tablets. Kindly fix this

1. You can see the container class has a width. That's why the remaining 375px. But not only this class. Please find which classes have a width and delete them. If the page is still not responsive kindly find which CSS makes happen that and fix it.

1. Background image should responsive too.

![image](https://user-images.githubusercontent.com/38635158/201053078-a50a95fa-b686-4118-8331-586fe6e82ef1.png)
___
After closed width
![image](https://user-images.githubusercontent.com/38635158/201054211-1b5aed86-f202-42aa-8520-eca9f263a1ea.png)